src/utilities/boleto/index.ts

```diff
@@ -134,3 +140,40 @@ export function format(boleto: string) {
       return result;
     }, '');
 }
+
+export function getValueInCents(digitableLine: string): number {
+  if (!digitableLine || !isValid(digitableLine)) return 0;
+
+  const digits = onlyNumbers(digitableLine);
+
+  const valueStartIndex = digits.length - 10;
+
+  return Number(digits.substr(valueStartIndex));
+}
+
+export function getExpirationDate(digitableLine: string): Date | null {
+  if (!digitableLine || !isValid(digitableLine)) return null;
+
+  const firstDay = new Date(1997, 9, 7);
+
+  const daysSinceFirstDay = digitableLine.substr(digitableLine.length - 14, 4);
```

Comment:
what do you think to put "14" and "4" in a variable and give a name to these numbers?